README.md

```diff
@@ -113,3 +118,5 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * [@Swag_iOS](https://twitter.com/Swag_iOS) for testing
 * [@jailbreakbuster](https://twitter.com/jailbreakbuster) for testing
```

Comment:
That’s not my Twitter handler. It’s @King4q1

Reply:
Updated and sorry